src/cart/src/cart/crt_context.c

```diff
@@ -426,6 +426,8 @@ crt_ctx_epi_abort(d_list_t *rlink, void *arg)
 	return rc;
 }
 
+extern uint32_t crt_swim_rpc_timeout;
```

Comment:
why not put in header?

Reply:
This is internal reference. I'd like to avoid to use this in other places. I'd prefer to use getter function instead. But in this place (internal place) for performance reason will be better to directly refer to this variable.

Reply:
moved to crt_internal.h